src/apimtemplate/Core/DeploymentDefinitions/Entities/SubscriptionDeploymentDefinition.cs

```diff
@@ -112,6 +112,11 @@ private string DependentApi()
 
 		private string DependentUser()
 		{
```

Comment:
@andrewdmoreno Can you please fix the indentation?

Reply:
Not sure how I missed this. Will fix now.

Reply:
Removed commit for version change.